src/app/app.config.ts

```diff
@@ -6,27 +6,49 @@ import { HttpClientModule } from '@angular/common/http';
 import { StudySetDevService } from './services/study-set-dev.service';
 import { initializeApp, provideFirebaseApp } from '@angular/fire/app';
 import { getAuth, provideAuth } from '@angular/fire/auth';
-import { getFirestore, provideFirestore } from '@angular/fire/firestore';
-import { getStorage, provideStorage } from '@angular/fire/storage';
-import { environment } from '../environments/environment.development';
+import { connectFirestoreEmulator, getFirestore, provideFirestore } from '@angular/fire/firestore';
+import { connectStorageEmulator, getStorage, provideStorage } from '@angular/fire/storage';
+import { environment } from '../environments/environment';
```

Comment:
why was this changed from environment.development to environment? The variable, useFirebase could be changed in environment.ts, but I think that would ignore the point of the development environment. It might've just been a change that was overlooked.

Reply:
The `environment` file gets replaced when the code is run in the development environment (i.e., ng serve) by the configs in `angular.json`
https://github.com/PonderTeam/ponder-app/blob/7dcd998da98c32b96830bafb87b77c0e53f80ce9/angular.json#L58-L66

If you run the code and look in the browser console, you should see "Angular is running in development mode."

## with `useFirebase: true`
<img width="1920" alt="image" src="https://github.com/PonderTeam/ponder-app/assets/4361319/d4da59b7-42ac-4c71-9eb8-fcf0cfa9dd3f">

## with `useFirebase: false`
<img width="1920" alt="image" src="https://github.com/PonderTeam/ponder-app/assets/4361319/5221be11-e059-485a-b333-ee3a28714a85">

Reply:
You can also click on the sets and confirm that the id's in the url's don't match the the id's in the cloud Firestore. Therefore, we can safely assume that one of the local data sources is being used instead.

Reply:
I see. I didn't know that. Thank you for clarifying.